2022/python/Day3.py

```diff
@@ -0,0 +1,123 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Created on Sun Dec 25 13:38:52 2022
+
+@author: raegoering
+
+--- Day 3: Rucksack Reorganization ---
+
+"""
+import string
+from itertools import islice
+
+def split_contents(contents_string, compartment):
```

Comment:
If you wanted, you could have this function return a tuple containing the front/back contents, then you wouldnt have to call it twice below!

Reply:
I know how much you love tuples ;)